DynamicAnalyzer/views/android.py

```diff
@@ -41,6 +42,186 @@
 def key(d, key_name):
     return d.get(key_name)
 
+def stopAVD(adb):
+    print "\n[INFO] Stopping MobSF Emulator"
+    try:
+        # adb -s emulator-xxxx emu kill
+        args = [adb, '-s', getIdentifier(), 'emu', 'kill']
+        subprocess.call(args)
+    except:
+        PrintException("[ERROR] Stopping MobSF Emulator")
+
+
+def deleteAVD(avd_path, avd_name):
+    print "\n[INFO] Deleting emulator files"
+    try:
+        config_file = os.path.join(avd_path, avd_name + '.ini')
+        if os.path.exists(config_file):
+            os.remove(config_file)
+
+        # TODO: Sometimes there is an error here because of the locks that avd does - check this out
+        avd_folder = os.path.join(avd_path, avd_name + '.avd')
+        if os.path.isdir(avd_folder):
+            shutil.rmtree(avd_folder)
+    except:
+        PrintException("[ERROR] Deleting emulator files")
+
+
+def duplicateAVD(avd_path, reference_name, dup_name):
+    print "\n[INFO] Duplicating MobSF Emulator"
+    try:
+        reference_ini = os.path.join(avd_path, reference_name + '.ini')
+        dup_ini       = os.path.join(avd_path, dup_name + '.ini')
+        reference_avd = os.path.join(avd_path, reference_name + '.avd')
+        dup_avd       = os.path.join(avd_path, dup_name + '.avd')
+
+        # Copy the files from the referenve avd to the one-time analysis avd
+        shutil.copyfile(reference_ini, dup_ini)
+        shutil.copytree(reference_avd, dup_avd)
+
+        # Replacing every occuration of the reference avd name to the dup one
+        for path_to_update in [
+            dup_ini,
+            os.path.join(dup_avd, 'hardware-qemu.ini'),
+            os.path.join(dup_avd, 'config.ini')
+        ]:
+            with open(path_to_update, 'r') as fd:
+                replaced_file = fd.read()
+                replaced_file = replaced_file.replace(reference_name, dup_name)
+            with open(path_to_update, 'w') as fd:
+                fd.write(replaced_file)
+    except:
+        PrintException("[ERROR] Duplicating MobSF Emulator")
+
+
+def startAVD(emulator, avd_name, emulator_port):
+    print "\n[INFO] Starting MobSF Emulator"
+    try:
+        args = [
+            emulator,
+            '-avd',
+            avd_name,
+            "-no-snapshot-save",
+            "-netspeed",
+            "full",
+            "-netdelay",
+            "none",
+            "-port",
+            str(emulator_port),
+        ]
+
+        if platform.system() == 'Darwin':
+            # There is a strage error in mac with the dyld one in a while.. this should fix it..
+            if 'DYLD_FALLBACK_LIBRARY_PATH' in os.environ.keys():
+                del os.environ['DYLD_FALLBACK_LIBRARY_PATH']
```

Comment:
From my tests, it will delete the env var only for the python run and will bot affect the general var, I will test it again